dcpy/models/product/dataset/metadata.py

```diff
@@ -40,23 +43,18 @@ class DatasetOverrides(BaseModel, extra="forbid"):
     display_name: str | None = None
     description: str | None = None
     tags: list[str] = []
-    destination_file_name: str | None = None
+    destination_file_name: str | None = None  # TODO filename
```

Comment:
looks like most of our code uses `filename` with no underscores, so we should standardize